app/main.py

```diff
@@ -2,0 +1,26 @@
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+from typing import List, Dict
+
+
+def go_to_cafe(friends: List[Dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.check_entry_requirements(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should have up-to-date vaccines"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
```

Comment:
According to the checklist, avoid comparing a variable to 0 if there is no need. You can simplify this condition to `if masks_to_buy:`.